.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimePeriodExtractor.cs

```diff
@@ -736,5 +756,30 @@ private Token MatchWithinNextPrefix(string subStr, string text, Token duration,
             return new Token(startOut, endOut);
         }
 
+        private List<Token> MatchPureNumberCases(string text, Token tok, bool before)

```

Comment:
What exactly is happening here? Are you taking advantage of the fact that a range with pure number is detected if at the end of the sentence? Is that it?
Can you add a comment to the method describing the assumption?

Reply:
I dropped the requirement that the pure number range must be at the end of sentence when it is contiguous to a time of day match. Will add comment.

Reply:
comment added